### PR TITLE
Fix #27 - Migration to Swift 4.2

### LINKED ISSUE #27: Please!! Convert your label to swift 4.2

### PR DESCRIPTION
**Pull request by teameh**
- Migration to Swift 4.2
- Updated sample project settings (enabled some warnings)
- Bump version

### CODE REVIEW

**Comment by teameh** [comment]
Haha build was still running ;)

File: Source/ContextLabel.swift [hunk 1]
```diff
@@ -135,18 +135,18 @@ open class ContextLabel: UILabel, NSLayoutManagerDelegate, UIGestureRecognizerDe
...
-    return .styleNone
+    return []
   }
   
   public var modifiedAttributedString: ((NSAttributedString) -> NSAttributedString)?
   
   public var didTouch: (TouchResult) -> Void = { _ in }
   
-  public var didCopy: (String!) -> Void = { _ in }
+  public var didCopy: (String?) -> Void = { _ in }
```
Comment (teameh) [inline Source/ContextLabel.swift]: @michaelloistl please check this change. I think it is okay, but it should be reviewed. See https://swift.org/blog/iuo/ for more info

Comment (michaelloistl) [reply Source/ContextLabel.swift]: Keeping the optional value as per your PR. Have also changed the delegate version of it.

Comment (teameh) [reply Source/ContextLabel.swift]: Ok!

**Pull request merged**